pkgs/build-support/ocaml/oasis.nix

```
@@ -0,0 +1,31 @@
+{ buildOcaml, ocaml_oasis }:
+
+{ name, version, buildInputs ? [], ...
```

The variables `name` and `version` are bound but never used. Is it on purpose?

---

You're right. lets change it to:

```suggestion
{buildInputs ? [], ...
```